### ISSUE TITLE
Weird Error Clashing with nut?

### ISSUE THREAD

**Issue by Salvora** [merdok/homebridge-webos-tv#210]
Hello,
I've set up homebridge-webos-tv quite some time now.
My tv doesnt support WOL so whenever I turn off my TV, I get webOS - Error: connect EHOSTUNREACH errors.
A day ago I've installed homebridge-nut to add my ups and for some reason the error I get from this has changed.
I don't know why but sometimes the errors of homebridge-nut appears on the webos.

Homebridge is running on Synology NAS docker with config:
```
        {
            "name": "LaxLGTV",
            "ip": "192.168.1.30",
            "mac": "3C:CD:93:C7:80:3B",
            "broadcastAdr": "C4:36:6C:AF:9C:04",
            "keyFile": "/homebridge/LaxLGTV.cookie",
            "prefsDir": "/homebridge/LaxLGTV",
            "pollingInterval": 5,
            "legacyTvService": false,
            "showInputButtons": true,
            "volumeControl": "true",
            "volumeLimit": 100,
            "channelControl": false,
            "mediaControl": false,
            "remoteSequenceButtons": false,
            "infoButtonAction": "INFO",
            "accessory": "webostv"
        },
```
     

Here is the error.
```
[3/13/2020, 1:21:04 AM] [LaxLGTV] webOS - Error: connect EHOSTUNREACH 192.168.1.30:3000
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1137:16)
    at module.exports.runLoopOnce (/homebridge/node_modules/homebridge-nut/node_modules/deasync/index.js:65:10)
    at module.exports (/homebridge/node_modules/homebridge-nut/node_modules/deasync-promise/index.js:13:9)
    at module.exports (/homebridge/node_modules/homebridge-nut/node_modules/system-sleep/module.js:24:36)
    at NutAccessory.getCheck (/homebridge/node_modules/homebridge-nut/index.js:164:5)
    at NutAccessory.servicePolling (/homebridge/node_modules/homebridge-nut/index.js:236:8)
    at Timeout._onTimeout (/homebridge/node_modules/homebridge-nut/index.js:238:10)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7) {
  errno: 'EHOSTUNREACH',
  code: 'EHOSTUNREACH',
  syscall: 'connect',
  address: '192.168.1.30',
  port: 3000
}
```

here is the error I normally get when TV is off.
[3/13/2020, 1:25:28 AM] [LaxLGTV] webOS - Error: connect ETIMEDOUT 192.168.1.30:3000
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1137:16) {
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '192.168.1.30',
  port: 3000
}
I want to ask what could be the problem?

**Comment by merdok** [comment]
I guess the homebridge-nut plugin is modifying or overwriting something. I cannot tell exactly why, but does this cause any problems for you?

**Comment by Salvora** [comment]
Yep, sometimes homebridge crashes. Sometimes it works ok on the logs but when I get to my phone all homebridge devices are unresponsive. I have removed the nut plug-in and had to clean install homebridge again.
What I don't get is I run it on homebridge docker and install all the plugins from config-ui-x so there shouldn't be any clashes or overwriting.
Anyway it was weird.

**Comment by merdok** [comment]
I guess this has something to do with your environment. I cannot think of anything else what is causing this.

**Comment by Salvora** [comment]
Thanks for the help.
I want to ask one thing. Do you have any plans to support TVs without wol feautre and create an optional parameter to arrange it. As you know without wol, you cannot wake TVs and homebridge throws errors since it cannot connect the TV. The only way is to use HDMI-CEC. I saw a fork that addresses this issue but it was pretty old.

**Comment by merdok** [comment]
wol has nothing to do with your the errors in your console, this is basically status checking which needs to be done in order for me to know when the tv gets turned on.

**Comment by Salvora** [comment]
> 
> 
> wol has nothing to do with your the errors in your console, this is basically status checking which needs to be done in order for me to know when the tv gets turned on.

No.
What I meant is, after I turn off my TV, like in 10 minutes, it cuts the power to the ethernet adapter too. So, basically when webos tries to get status, it cannot connect and throws "connect ETIMEDOUT". This is normal.
In this state, you normally cannot turn on TV with homebridge or with any other network device. The only way is to use a HDMI device.
The fork I was talking about is this one.
https://github.com/mrbalintlorand/homebridge-webos3
It is old though.

**Comment by merdok** [comment]
Oh, but this requires the TV to be connected via HDMI to the raspberry pi, right?

BTW, what TV do you have that does not support wol?

**Comment by Salvora** [comment]
(: 
It is LG TV 32LF630V - ZA.
https://www.displayspecifications.com/en/model/7e791d2
It was originally webos 2.0 but later they published webos 3.0 update. But all the TVs in this series should also not have WOL.
> Oh, but this requires the TV to be connected via HDMI to the raspberry pi, right?

Not sure about this one. But any HDMI device with network stable connectivity should be enough. For example, I have Chromecast ultra attached to the HDMI. If I play something on it, even if the TV is off, it can turn on TV. However, this is not an elegant solution. ):

**Comment by merdok** [comment]
Got it 👍
